src/widgets/splits/SplitHeader.cpp

```diff
@@ -544,6 +550,7 @@ void SplitHeader::updateChannelText()
 
     if (auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get()))
     {
+        twitchChannel->refreshTitle();
```

Comment:
this is the place that I don't want it to be called from, cuz this will be fired 4 times in quick succession when the stream is live and we get new game/title/viewer/live status data